src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        projects = {
+            d['id']: d
```

Comment:
Just use the `d.id` value here to prevent having to do the weird type conversion on below (same for `users`.)